src/components/PrimaryNav/_primary-nav.scss

```diff
@@ -0,0 +1,57 @@
+@import "vanilla-framework/scss/settings_colors";
+
+.p-primary-nav {
+  a,
+  a:hover {
+    color: $color-light;
+  }
+
+  &__header {
+    display: grid;
+    grid-template-columns: 1fr 40px;
+    margin-bottom: -1rem;
+    padding: 0.5rem;
+  }
+
+  &__toggle {
+    background: none;
+    border: 0;
+    transition: transform 0.25s ease-in-out;
+
+    &:hover {
+      background: 0;
+    }
+  }
+
+  .p-list__link {
+    color: $color-light;
+  }
+
+  .p-list__item {
+    padding-left: 2.5rem;
```

Comment:
It would be nicer to add the padding too the links so you can click the row and now just on the word.